basics/src/test/java/ConvertibleCarTest.java

```diff
@@ -0,0 +1,44 @@
+import static org.junit.Assert.*;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ConvertibleCarTest {
+	private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();

```

Comment:
This may be a little overengineered. Streams may require closing.

https://mikemybytes.com/2021/01/26/closing-java-streams-with-autocloseable/